usefulUtilities/marketplaceItemAdvertiser/README.md

```diff
@@ -0,0 +1,28 @@
+# Marketplace Item Advertiser
+When this script is attached to an entity as an entity script, the entity serves as an "Marketplace Item Advertiser".
+
+## Features
+- When a user clicks with their mouse or uses the red hand controller laser on the entity that is running this script, the user will be presented with the Marketplace Checkout page associated with the Marketplace Item ID that is referenced in the `userData` of that entity.
+
+## Setup
+1. Add an entity to your domain.
+    - A Box primitive would work perfectly.
+    - The entity should have some affordance to indicate that a user can click on the entity. A billboard with text "CLICK HERE TO PURCHASE XYZ APP" is one example.
+    - Note that anyone who can modify the `userData` of this entity will be able to control which Checkout page that users see!
+2. Add the below `userData` object to the attached entity's `userData`
+    1. Fill in the `marketplaceID` value in the `userData` by adding Marketplace IDs to Items that you want users to purchase.

```

Comment:
I would change the "1." to a " - ". 

In the markup it shows up as an "i." tag which is different from the "-" syntax for step 1.

Reply:
this is on purpose :) it's a substep rather than an informational step (as in step 1)